card/mod/pointer/set/abstract/02_pointer/edit.rb

```diff
@@ -27,7 +27,7 @@ def add_item_button
   end
 
   def edit_list_item item
-    haml :list_item, pointer_item: item, options_card: options_card_name
+    haml :list_item, item: item, options_card: options_card_name
```

Comment:
this was a screwup of mine from the view args cleanup. breaks all pointer editing.